pkg/scheduler/framework/arguments.go

```diff
@@ -0,0 +1,28 @@
+package framework
+
+import (
+	"strconv"
+
+	"github.com/golang/glog"
+)
+
+type Arguments map[string]string
+
+func (a Arguments) GetInt(ptr *int, key string) {
```

Comment:
return value instead of pointer.

Reply:
so, I think the definition should be.
```
var ERRNOTFOUND=errors.New("ARGUMENT NOT FOUND")

func (a Arguments) GetInt(key string) (value int, err error) { ... }
```
when the value of this key is not found, the return value should be 0, ERRNOTFOUND
when the value is in invalid format of int, the return value should be 0, err of parsing